Rakefile

```diff
@@ -174,3 +181,61 @@ task :validate do
     exit!
   end
 end
+
+desc 'Build changlog'
+task :changelog do
+  [latest_changes_file, history_file].each do |f|
+    unless File.exists?(f)
+      puts "#{f} does not exist but is required to build a new release."
+      exit!
+    end
+  end
+  
+  latest_changes = File.open(latest_changes_file)
+  version_pattern = "# #{version}"
+  
+  if !`grep "#{version_pattern}" #{history_file}`.empty?
+    puts "#{version} is already described in #{history_file}"
+    exit!
+  end
+
+  begin
+    unless latest_changes.readline.chomp! =~ %r{#{version_pattern}}
+      puts "#{latest_changes_file} should begin with '#{version_pattern}'"
+      exit!
+    end
+  rescue EOFError
+    puts "#{latest_changes_file} is empty!"
+    exit!
+  end
+  
+  body = latest_changes.read
+  body.scan(/\s*#\s+\d\.\d.*/) do |match|
+    puts "#{latest_changes_file} may not contain multiple markdown headers!"
+    exit!
+  end
+  
+  temp = Tempfile.new
+  temp.puts("#{version_pattern} / #{date}\n#{body}\n")
+  temp.close
+  `cat #{history_file} >> #{temp.path}`
+  `cat #{temp.path} > #{history_file}`
+end
+
+desc 'Precompile assets'
+task :precompile do
```

Comment:
`gollum-lib` does not contain any assets and doesn't contain `sprockets` as a dependency. Does this need to be in the `gollum-lib` Rakefile?

Reply:
This was copied from the `gollum` Rakefile in one of the commits that's already in `master`. I'll get rid of it in a separate PR!